cmd/osmosisd/cmd/testnetify/cmd.go

```diff
@@ -62,5 +62,9 @@ func writeGenesis(genesis app.GenesisState, path string) error {
 	}
 	defer file.Close()
 	encoder := json.NewEncoder(file)
-	return encoder.Encode(genesis)
+	err = encoder.Encode(genesis)
+	if err != nil {
+		return err
+	}
```

Comment:
Why is it changed from a shorter implementation to a longer one?

Reply:
I don't know...  I think my linter was demanding this for some reason.  Let me try putting it back to how it was